docs/source/cli/poet.rst

```diff
@@ -19,23 +19,41 @@
 PoET CLI
 ********
 
+The PoET CLI initializes the proof-of-concept consensus mechanism for
+Sawtooth by generating enclave setup information and creating a Batch
+for the genesis block. For more information, see :doc:/architecture/poet.
+
+The poet command generates information for configuring a node to use
```

Comment:
Delete this paragraph.  Sorry - that was my mistake in the google doc (it belongs only in the next "poet" subsection).